language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeIT.java

```diff
@@ -18,142 +18,122 @@
   }
 
-  @Test public void analyzeSentiment_returnNegativeFile() throws Exception {
-    // Act
-    Sentiment sentiment =
-        analyzeApp.analyzeSentimentFile("gs://" + BUCKET + "/natural-language/"
+  @Test
+  public void analyzeSentiment_returnNegative() throws Exception {
+    Sentiment sentiment = Analyze.analyzeSentimentFile("gs://" + BUCKET + "/natural-language/"
         + "sentiment/bladerunner-neg.txt");
-
-    // Assert
-    assertThat((double)sentiment.getMagnitude()).isGreaterThan(0.0);
-    assertThat((double)sentiment.getScore()).isLessThan(0.0);
+    assertThat(sentiment.getMagnitude()).isGreaterThan(0.0F);
+    assertThat(sentiment.getScore()).isLessThan(0.0F);
   }
 
-  @Test public void analyzeSentiment_returnNeutralFile() throws Exception {
-    // Act
-    Sentiment sentiment =
-        analyzeApp.analyzeSentimentFile("gs://" + BUCKET + "/natural-language/"
+  @Test
+  public void analyzeSentiment_returnNeutralFile() throws Exception {
+    Sentiment sentiment = Analyze.analyzeSentimentFile("gs://" + BUCKET + "/natural-language/"
         + "sentiment/bladerunner-neutral.txt");
-
-    // Assert
-    assertThat((double)sentiment.getMagnitude()).isGreaterThan(1.0);
-// TODO sentiment score for netural sample appears to be zero now.
-//    assertThat((double)sentiment.getScore()).isGreaterThan(0.0);
+    assertThat(sentiment.getMagnitude()).isGreaterThan(1.0F);
+    // TODO sentiment score for netural sample appears to be zero now.
+    // assertThat((double)sentiment.getScore()).isGreaterThan(0.0);
   }
 
-  @Test public void analyzeSyntax_partOfSpeech() throws Exception {
-    // Act
-    List<Token> token =
-        analyzeApp.analyzeSyntaxText(
-            "President Obama was elected for the second term");
+  @Test
+  public void analyzeSyntax_partOfSpeech() throws Exception {
+    List<Token> tokens = Analyze
+        .analyzeSyntaxText("President Obama was elected for the second term");
 
-    List<Tag> got = token.stream().map(e -> e.getPartOfSpeech().getTag())
+    List<Tag> got = tokens.stream().map(e -> e.getPartOfSpeech().getTag())
         .collect(Collectors.toList());
 
-    // Assert
     assertThat(got).containsExactly(Tag.NOUN, Tag.NOUN, Tag.VERB,
         Tag.VERB, Tag.ADP, Tag.DET, Tag.ADJ, Tag.NOUN).inOrder();
   }
 
-  @Test public void analyzeSyntax_partOfSpeechFile() throws Exception {
-    // Act
-    List<Token> token =
-        analyzeApp.analyzeSyntaxFile("gs://" + BUCKET + "/natural-language/"
-                + "sentiment/bladerunner-neutral.txt");
+  @Test
+  public void analyzeSyntax_partOfSpeechFile() throws Exception {
+    List<Token> token = Analyze.analyzeSyntaxFile("gs://" + BUCKET + "/natural-language/"
+        + "sentiment/bladerunner-neutral.txt");
 
     List<Tag> got = token.stream().map(e -> e.getPartOfSpeech().getTag())
         .collect(Collectors.toList());
 
-    // Assert
     assertThat(got).containsExactly(Tag.PRON, Tag.CONJ, Tag.VERB, Tag.CONJ, Tag.VERB,
```

Comment:
This will make us unhappy eventually.

Reply:
I'm hoping POS tags have less likelihood of making us unhappy.